<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aboutDialog</class>
 <widget class="QDialog" name="aboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>328</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <widget class="QLabel" name="aboutLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>361</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head&gt;&lt;title&gt;About&lt;/title&gt;&lt;/head&gt;&lt;body&gt;&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:700;&quot;&gt;YT Channel Downloader&lt;/span&gt;&lt;/h3&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Version:&lt;/span&gt; 0.4.12&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Years: &lt;/span&gt;2023-2025&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;License: &lt;/span&gt;MIT &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Github: &lt;/span&gt;&lt;a href=&quot;https://github.com/hyperfield/yt-channel-downloader&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;View on Github&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Author: &lt;/span&gt;&lt;a href=&quot;mailto:info@quicknode.net&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;hyperfield&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Contributors: &lt;/span&gt;&lt;p&gt;
  &lt;a href=&quot;https://github.com/dsasmblr&quot; style=&quot;text-decoration: underline; color: #0000ff;&quot;&gt;dsasmblr&lt;/a&gt;, 
  &lt;a href=&quot;https://github.com/djfm&quot; style=&quot;text-decoration: underline; color: #0000ff;&quot;&gt;djfm&lt;/a&gt;, 
  &lt;a href=&quot;https://github.com/arvinnick&quot; style=&quot;text-decoration: underline; color: #0000ff;&quot;&gt;arvinnick&lt;/a&gt;
&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutOkButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/icon.png&quot; width=&quot;64&quot; height=&quot;64&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
